### DISCUSSION TITLE
build failure with Ant-Media-Enterprise due to maven srt dependency

### DISCUSSION THREAD

**Discussion by hd51515** [ant-media/Ant-Media-Server#5117] [General]
followed the instructions here on Ubuntu 18.04 https://antmedia.io/docs/guides/advanced-usage/building-ams-from-source-code/
Same result on mater, 2.5.3 and 2.6.0 branches

[INFO] ------------< io.antmedia.enterprise:ant-media-enterprise >-------------
[INFO] Building ant-media-enterprise 2.6.0
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for org.bytedeco:srt-platform:jar:1.5.1-1.5.8 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.600 s
[INFO] Finished at: 2023-05-17T13:00:08-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project ant-media-enterprise: Could not resolve dependencies for project io.antmedia.enterprise:ant-media-enterprise:jar:2.6.0: Failure to find org.bytedeco:srt-platform:jar:1.5.1-1.5.8 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

The following was attempted to fix but it still results in the same error as above
 wget https://storage.sbg.cloud.ovh.net/v1/AUTH_8cb28f9bc6ee43f0a3a1825efbb4311e/test-storage/srt-all.zip
 rm -rf ~/.m2/repository/org/bytedeco/srt
 rm -rf ~/.m2/repository/org/bytedeco/srt-platform
 unzip srt-all.zip
 mv srt ~/.m2/repository/org/bytedeco
 mv srt-platform ~/.m2/repository/org/bytedeco

**Comment by mekya** [comment]
Hi @hd51515,

SRT dependency is a built that is prepared by Ant Media. You can access it via this link https://storage.sbg.cloud.ovh.net/v1/AUTH_8cb28f9bc6ee43f0a3a1825efbb4311e/test-storage/srt-all.zip
This link may change.

Secondly, let me what you are trying to achieve by building the enterprise from source code. If we know,  we may help you better.

Regards
Oguz

**Reply by hd51515** [reply]
The following was attempted to fix but it still results in the same error as above
wget https://storage.sbg.cloud.ovh.net/v1/AUTH_8cb28f9bc6ee43f0a3a1825efbb4311e/test-storage/srt-all.zip
rm -rf ~/.m2/repository/org/bytedeco/srt
rm -rf ~/.m2/repository/org/bytedeco/srt-platform
unzip srt-all.zip
mv srt ~/.m2/repository/org/bytedeco
mv srt-platform ~/.m2/repository/org/bytedeco

**Reply by mekya** [reply]
There may be some kind of another issue on your side because these structure are being used in the automation.
If you give some more context information about your system such as OS java version, etc. It may help resolve the problem.

Furthermore, you can join to this event and we can take a look at your issue after the event. 
https://github.com/orgs/ant-media/discussions/5121

**Comment by hd51515** [comment]
I want to add some debug message in webrtcforwarder, sfuforwarder to understand the current code/flow

**Reply by hd51515** [reply]
uname -a
Linux 5.4.0-148-generic #165~18.04.1-Ubuntu SMP Thu Apr 20 01:14:06 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

java -version
openjdk version "11.0.19" 2023-04-18
OpenJDK Runtime Environment (build 11.0.19+7-post-Ubuntu-0ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.19+7-post-Ubuntu-0ubuntu118.04.1, mixed mode, sharing)
